lib/oli_web/templates/layout/chromeless.html.eex

```diff
@@ -36,6 +36,7 @@
 
     <!-- Janus Part Component external CSS Files -->
     <link rel="stylesheet" href="/css/janus_image_carousel_delivery.css"/>
+    <link id="vendor-css" rel="stylesheet" href="/css/vendor.css" />
```

Comment:
Should these be using Routes.static_path ?

Reply:
Probably, but idt its a huge deal either way since its unlikely we would move the static assets location. Maybe a good idea for consistency if nothing else

Reply:
That's what I figured.

One use-case of moving the static assets would be plopping them on a CDN to speed things up a bit, so if we ever get there we should audit all the links / scripts